### PR TITLE
MS-1387 task: automate deployments to dev

### PR DESCRIPTION
**Pull request by jjrobothamho**
- update drone.yml
- incorporate deployments to dev in drone pipeline

### CODE REVIEW

**Comment by jjrobothamho** [comment]
Test to see if our pipeline will automate a deployment to dev when the master branch has been built successfully.

File: .drone.yml [hunk 1]
```diff
@@ -61,6 +61,14 @@ pipeline:
       branch: master
       event: push
 
+    trigger_deploy_to_dev:
+      image: quay.io/ukhomeofficedigital/drone-trigger:v0.3.0
+      drone_server: https://drone-gitlab.acp.homeoffice.gov.uk/modern-slavery/kube-modern-slavery
```
Comment (oyelekci) [inline .drone.yml]: drone server should be https://drone-gitlab.acp.homeoffice.gov.uk - not the full repo path

Comment (oyelekci) [reply .drone.yml]: you also need to reference the relevant secret for the gitlab drone token

Comment (oyelekci) [reply .drone.yml]: I'd also add when this should triggered, e.g.
```
when:
      branch: master
      event: push
```

File: .drone.yml [hunk 2]
```diff
@@ -61,6 +61,19 @@ pipeline:
       branch: master
       event: push
 
+    trigger_deploy_to_dev:
+      image: quay.io/ukhomeofficedigital/drone-trigger:v0.3.0
+      drone_server: https://drone-gitlab.acp.homeoffice.gov.uk
+      repo: modern-slavery/kube-modern-slavery
+      secrets:
+        - drone_token
```
Comment (jjrobothamho) [inline .drone.yml]: Added the secret on dev environment. Hope this is what you meant.

File: documents/release-guidelines.md [hunk 3]
```diff
@@ -59,6 +59,8 @@ The above commit will also need to be deployed. See [Deployment for a Release](#
 ### Deployment for Development Testing <a name="dev-deployment"></a>
 If you are tesing issues for a release you must deploy the latest commit with the targeted issues to the developer environment. See [Deployment](#deployment)
 
+On this service if you have merged any pull requests to the master branch your changes will be automatically deployed to the development environment.
```
Comment (jjrobothamho) [inline documents/release-guidelines.md]: What do think Sulthan?

File: .drone.yml (hunk 2 above)
Comment (oyelekci) [reply .drone.yml]: The way you refer to it here is fine but the secret itself doesn't appear to be in Drone.

File: documents/release-guidelines.md (hunk 3 above)
Comment (sulthan-ahmed) [reply documents/release-guidelines.md]: looks good 👍

Comment (sulthan-ahmed) [reply documents/release-guidelines.md]: 👍

File: .drone.yml (hunk 2 above)
Comment (jjrobothamho) [reply .drone.yml]: Thanks, I have added it to the drone repo.

**Pull request merged**